hrapplications/models.py

```diff
@@ -27,6 +27,7 @@ def __str__(self):
 
 
 class HRApplicationComment(models.Model):
+    created_on = models.DateTimeField(auto_now_add=True)
```

Comment:
I'm unable to evolve the model changes without adding `null=True` to the `created_on` field. For compatibility I'd recommend you add this to the pull request.